lib/backend/k8s/k8s.go

```diff
@@ -739,9 +741,85 @@ func (c *KubeClient) deleteGlobalConfig(k *model.KVPair) error {
 }
 
 func (c *KubeClient) getHostConfig(k model.HostConfigKey) (*model.KVPair, error) {
+	if k.Name == "IpInIpTunnelAddr" {
+		n, err := c.clientSet.Nodes().Get(k.Hostname, metav1.GetOptions{})
+		if err != nil {
+			return nil, resources.K8sErrorToCalico(err, k)
+		}
+
+		tunIp, err := getTunIp(n)
+		if err != nil {
+			return nil, err
+		}
+
+		return &model.KVPair{Key: k, Value: tunIp}, nil
+	}
+
 	return nil, errors.ErrorResourceDoesNotExist{Identifier: k}
 }
 
 func (c *KubeClient) listHostConfig(l model.HostConfigListOptions) ([]*model.KVPair, error) {
-	return []*model.KVPair{}, nil
+	var kvps = []*model.KVPair{}
+
```

Comment:
Or an empty string as it could be looking to list all info for all hosts or one host, I've now set it up to ignore anything other than `"" || "IpInIpTunnelAddr"`